documentation/book/proc-upgrading-kafka-connect-resources.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='proc-upgrade-kafka-connect-resources-{context}']
+= Upgrading Kafka Connect resources
+
+.Prerequisites
+
+* The Cluster Operator is up and running.
```

Comment:
A 0.12.0 or later CO is up and running.

I realise this is difficult because of upstream and downstream versions, so maybe you can reword it "A Cluster Operator which supports v1beta1 API version is up and running"